mk/PcapPlusPlus.mk.macosx.arm64

```diff
@@ -0,0 +1,11 @@
+### MAC OS X ###
+
+# includes
+PCAPPP_INCLUDES += -I$(MACOS_SDK_HOME)/usr/include/netinet
+
+# libs
+PCAPPP_LIBS += -lpcap -lpthread -framework SystemConfiguration -framework CoreFoundation
```

Comment:
I am not really happy with the code duplication here. 
( Lines copied from here : https://github.com/seladb/PcapPlusPlus/blob/master/mk/PcapPlusPlus.mk.macosx#L3-L7 )
Could we / Should we include `PcapPlusPlus.mk.macosx` instead?